### PR TITLE
Modifications per code review and Strava tour type mapping enhancement

### PR DESCRIPTION
**Pull request by FJBDev**
- Modifications and reverts per previous [PR](https://github.com/wolfgang-ch/mytourbook/pull/360) code review
- Enhanced the Strava tour type mapping [per user request](https://sourceforge.net/p/mytourbook/discussion/622811/thread/e07bd4d3d7/#2e74/6d32/cbef/d00a)

### CODE REVIEW

File: bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+cleanup.add_all=false
```
Comment (FJBDev) [inline bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs:1]: @wolfgang-ch 
I have updated this file and it resolved all the non desired code optimizations that were added [in 2021.06 (4.20)](https://www.eclipse.org/eclipse/news/4.20/jdt.php#use-instanceof)

**Comment by wolfgang-ch** [comment]
Now I'm happy with this PR :-)

File: bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs (hunk 1 above)
Comment (wolfgang-ch) [reply bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs:1]: You have to update this file in all MT plugins, see commit comment in https://github.com/wolfgang-ch/mytourbook/commit/72b6585e541c078ec4bad6a69f7ebedf9da1e1ba how to do it easily

Comment (FJBDev) [reply bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs:1]: Ok, i thought this file was used by all the other plugins. I now know it for the next time Eclipse adds cleanups.

**Pull request merged**